web/src/modules/feedback/components/FeedbackBox.js

```diff
@@ -75,8 +81,12 @@ export class FeedbackBox extends React.PureComponent<PropsType> {
           commentMessage={isPositiveRatingSelected ? t('positiveComment') : t('negativeComment')}
           onCommentChanged={onCommentChanged}
           required={!isPositiveRatingSelected} />
+        <Description>{t('contactMailAdress')}</Description>
+        <TextInput
+          onChange={onContactMailChanged}
+        />
```

Comment:
Can't we reuse `FeedbackComment` for this?

Reply:
Imo comment and contactMail should be seperated because they just are two different entries.

Reply:
@klinzo so what should we do?

Reply:
I think you could refactor `FeedbackComment` to sth like `FeedbackText` or `FeedbackEntry` (any name that fits)